ports/rlottie/portfile.cmake

```
@@ -0,0 +1,27 @@
+set(VCPKG_POLICY_SKIP_ABSOLUTE_PATHS_CHECK enabled)
+set(SOURCE_PATH ${CURRENT_BUILDTRESS_DIR})
+
+vcpkg_from_github(
+        OUT_SOURCE_PATH SOURCE_PATH
+        REPO Samsung/rlottie
+        REF e3026b1e1a516fff3c22d2b1b9f26ec864f89a82
+        SHA512 3b9985606d9c475e77ecb018cfe65cde1170f10e9d2c3e18b60178d3954a4870e5141aa06bb79e803fcdbcf98742bcf72a359625a3b1409125ec3a4a1b0126c4
+        PATCHES
+            0001-export-runtime.patch
+)
+
+vcpkg_cmake_configure(
+        SOURCE_PATH "${SOURCE_PATH}"
+        OPTIONS_DEBUG
+            -DLIB_INSTALL_DIR="${CURRENT_PACKAGES_DIR}/debug/lib"
+        OPTIONS_RELEASE
+            -DLIB_INSTALL_DIR="${CURRENT_PACKAGES_DIR}/lib"

```

According to the patch, this should take relative paths.
And quoting style is not portable.
```suggestion
        OPTIONS
            -DLIB_INSTALL_DIR=lib
```